### DISCUSSION TITLE
RuntimeError: Given groups=1, weight of size [512, 1, 10], expected input[1, 5945, 1] to have 1 channels, but got 5945 channels instead

### DISCUSSION THREAD

**Discussion by chze99** [fishaudio/Bert-VITS2#303] [Q&A]

**Comment by chze99** [comment]
from what i tried seem like it happen when i got >4 wav file at dataset

**Comment by todochenxi** [comment]
Hi, if you set batch_size =1, an error may be reported

**Comment by nqbefgvs** [comment]
同楼上，我在python webui_process.py弹出的网页里面调整了批大小(Batch size)改成4之后重新执行步骤一到四，就能正常训练了。我猜测是跟显存大小有关系，批大小参数的描述里写着“24 GB 显存可用 12”，我的电脑本地的显存只有8g，我改成了4就没有报错了。
